Sdl.Community.GroupShareKit/Clients/IProjectClient.cs

```diff
@@ -613,15 +613,16 @@ public interface IProjectClient
 		Task UndoCheckout(string projectId, string languageFileId);
 
 		///  <summary>
-		/// Health check call used to keep the OE license seat taken
+		/// Health check call used to keep the OE license advanced/basic seat taken
+        /// By default, if no parameter advanced/basic is passed in api route, advanced mode is checkout.
 		///  </summary>
 		///  This method requires authentication.
 		///  See the <a href="http://gs2017dev.sdl.com:41234/documentation/api/index#/">API documentation</a> for more information.
 		///  <exception cref="AuthorizationException">
 		///  Thrown when the current user does not have permission to make the request.
 		///  </exception>
 		///  <exception cref="ApiException">Thrown when a general API error occurs.</exception>
-		Task<string> OnlineCheckoutHealthCheck();
+		Task<string> OnlineCheckoutHealthCheck(string editorProfileMode);
```

Comment:
I was thinking that maybe an enum with the editor mode will be handy for the user, and in that case we'll make sure they'll not enter a wrong string.